readthedocs/core/middleware.py

```diff
@@ -57,15 +57,15 @@ def process_request(self, request):
                 request.session = SessionBase()  # create an empty session
                 return
 
-        if settings.SESSION_COOKIE_SAMESITE:
-            super().process_request(request)
-        else:
+        if settings.SESSION_COOKIE_SAMESITE == "None":
```

Comment:
It seems the logic is inverted here now. Before, when the setting was `None` we were calling `super()` but now we are running the code inside the if.

```
>>> if None:
...   print("None")
... 
>>> 
```

Reply:
The else statement was moved to the if, so the logic remains the same.